lib/ansible/modules/cloud/azure/azure_rm_containerregistry_facts.py

```diff
@@ -0,0 +1,221 @@
+version_added: "2.7"
+short_description: Get Registry facts.
+description:
+    - Get facts of Registry.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group to which the container registry belongs.
+        required: True
+    name:
+        description:
+            - The name of the container registry.
+    tags:
+        description:
+            - Limit results by providing a list of tags. Format tags as 'key' or 'key:value'.
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Get instance of Registry
+    azure_rm_containerregistry_facts:
+      resource_group: sampleresourcegroup
+      name: sampleregistry
+
+  - name: List instances of Registry
+    azure_rm_containerregistry_facts:
+      resource_group: sampleresourcegroup
+'''
+
+RETURN = '''
+registries:
+    description: A list of dictionaries containing facts for registries.
+    returned: always
+    type: complex
+    contains:
+        id:
+            description:
+                - The resource ID.
+            returned: always
+            type: str
+            sample: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.ContainerRegistry/registr
+                    ies/myRegistry"
+        name:
+            description:
+                - The name of the resource.
+            returned: always
+            type: str
+            sample: myRegistry
+        location:
+            description:
+                - The location of the resource. This cannot be changed after the resource is created.
+            returned: always
+            type: str
+            sample: westus
+        admin_user_enabled:
+            description:
+                - Is admin user enabled.
+            returned: always
+            type: bool
+            sample: yes
+        sku:
+            description:
+                - The SKU name of the container registry.
+            returned: always
+            type: str
+            sample: Classic
+        provisioning_state:
+            description:
+                - Provisioning state of the container registry
+            returned: always
+            type: str
+            sample: Succeeded
+        login_server:
+            description:
+                - Login server for the registry.
+            returned: always
+            type: str
+            sample: acrd08521b.azurecr.io
+'''
+
+from ansible.module_utils.azure_rm_common import AzureRMModuleBase
+
+try:
+    from msrestazure.azure_exceptions import CloudError
+    from msrestazure.azure_operation import AzureOperationPoller
+    from azure.mgmt.containerregistry import ContainerRegistryManagementClient
+    from msrest.serialization import Model
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+
+class AzureRMRegistriesFacts(AzureRMModuleBase):
```

Comment:
Pls use full service name: ContainerRegsitry. and I found actually older facts are not with "s"